internal/service/dns_config/model_config_view.go

```diff
@@ -0,0 +1,592 @@
+			types.ObjectValueMust(ConfigACLItemAttrTypes, map[string]attr.Value{
+				"access":   types.StringValue("allow"),
+				"acl":      types.StringNull(),
+				"address":  types.StringNull(),
+				"element":  types.StringValue("any"),
+				"tsig_key": types.ObjectNull(ConfigTSIGKeyAttrTypes),
+			}),
+		})),
+		MarkdownDescription: `Optional. Specifies which clients have access to the view.  Defaults to empty.`,
+	},
+	"match_destinations_acl": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: ConfigACLItemResourceSchemaAttributes,
+		},
+		Optional: true,
+		Computed: true,
+		Default: listdefault.StaticValue(types.ListValueMust(types.ObjectType{AttrTypes: ConfigACLItemAttrTypes}, []attr.Value{
+			types.ObjectValueMust(ConfigACLItemAttrTypes, map[string]attr.Value{
+				"access":   types.StringValue("allow"),
+				"acl":      types.StringNull(),
+				"address":  types.StringNull(),
+				"element":  types.StringValue("any"),
+				"tsig_key": types.ObjectNull(ConfigTSIGKeyAttrTypes),
+			}),
+		})),
+		MarkdownDescription: `Optional. Specifies which destination addresses have access to the view.  Defaults to empty.`,
+	},
+	"match_recursive_only": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: `Optional. If _true_ only recursive queries from matching clients access the view.  Defaults to _false_.`,
+	},
+	"max_cache_ttl": schema.Int64Attribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             int64default.StaticInt64(604800),
+		MarkdownDescription: `Optional. Seconds to cache positive responses.  Unsigned integer, min 1 max 604800 (7d).  Defaults to 604800 (7d).`,
+	},
+	"max_negative_ttl": schema.Int64Attribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             int64default.StaticInt64(10800),
+		MarkdownDescription: `Optional. Seconds to cache negative responses.  Unsigned integer, min 1 max 604800 (7d).  Defaults to 10800 (3h).`,
+	},
+	"max_udp_size": schema.Int64Attribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             int64default.StaticInt64(1232),
+		MarkdownDescription: `Optional. _max_udp_size_ represents maximum UDP payload size. The maximum number of bytes a responding DNS server will send to a UDP datagram.  Defaults to 1232 bytes.`,
+	},
+	"minimal_responses": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: `Optional. When enabled, the DNS server will only add records to the authority and additional data sections when they are required.  Defaults to _false_.`,
+	},
+	"name": schema.StringAttribute{
+		Required:            true,
+		MarkdownDescription: `Name of view.`,
```

Comment:
Can you also add a test for `name`. The update case shouldn't fail. If it fails, you will require a replace of resource if the name is changed. E.g. 

https://github.com/infobloxopen/terraform-provider-bloxone/blob/35d0e1d2ace37835ce215edd00b8371606f41499/internal/service/ipam/model_ipamsvc_address.go#L148-L150

Reply:
Resolved